src/main/kotlin/lotto/controller/LottoController.kt

```diff
@@ -13,10 +13,11 @@ fun main() {
 }
 
 class LottoController(private val inputView: InputView, private val resultView: ResultView) {
-    fun purchase() : Tickets {
+    fun purchase(): Tickets {
         val purchaseAmount = inputView.enterPurchaseAmount()
-        val tickets = LottoShop().purchase(purchaseAmount)
-        resultView.printTickets(tickets)
+        val manualTickets = inputView.enterManualTickets()
+        val tickets = LottoShop().purchase(purchaseAmount, manualTickets)
```

Comment:
When we purchase the tickets, a new instance of `LottoShop` is created each time.
Do you think that's necessary? How else could we handle that?

Here’s a reference you might find helpful:
- [[Kotlin Docs] Object declarations and expressions](https://kotlinlang.org/docs/object-declarations.html)

---

(+) Similarly, `LottoNumber` is a value object with a fixed range of values.
It might be worth thinking about how instances are managed.
I recommend revisiting what we covered in the last feedback session!